http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java

```diff
@@ -95,6 +112,9 @@ public void vertxHttpClientWithNameParameter() {
                     assertThat("Body response", body.toString().contains("Hello, Bender!"));
                 })
                 .onFailure(Throwable::printStackTrace);
+
+        httpClient.close();
```

Comment:
Advantage of try-with-resources and executing this in `finally` would be that this line would also be executed when tests failed. Anyway, this is already an improvement, let's keep this for now.